### ISSUE TITLE
Update Vote page to match Cruella's model

### ISSUE THREAD

**Issue by bcstryker** [pickle-finance/pickle-ui#520]
- [ ] Add delegate vote functionality and UI components to voting page
- [ ] Link to voting from DILL page

![Screen Shot 2022-05-31 at 8 51 35 PM](https://user-images.githubusercontent.com/24856006/171306684-b2eec09b-84ac-4a3a-94c1-298b5f18cf79.png)

**Comment by bcstryker** [comment]
Most of this issue was already completed. Remaining task is blocked by lack of delegation capabilities I believe. Waiting on something to happen on that front, will open new issue to adjust vote page once that functionality is available
